modules/controller.nix

```diff
@@ -8,6 +8,12 @@ let
     mkOption
     types
   ;
+
+  # Turn it off and on again
+  steampad-controller-reset = pkgs.writeShellScript "steampad-controller-reset" ''
+    echo 0 > /sys/devices/pci0000:00/0000:00:08.1/0000:04:00.4/usb3/3-3/authorized
+    echo 1 > /sys/devices/pci0000:00/0000:00:08.1/0000:04:00.4/usb3/3-3/authorized
```

Comment:
AFAICT this path should be safe to hardcode. If it ends up causing issues, we'll have to act on the pid and vid `28de:1205`, which is not exactly trivial to do without more scripting.

```
Bus 003 Device 002: ID 28de:1205 Valve Software Steam Controller
```